components/widgets/modals/full_screen_modal.tsx

```diff
@@ -29,7 +29,7 @@ class FullScreenModal extends React.Component<Props> {
     public componentDidMount() {
         document.addEventListener('keydown', this.handleKeypress);
         document.addEventListener('focus', this.enforceFocus, true);
-        this.resetFocus();
+        this.enforceFocus();
```

Comment:
`resetFocus` was unconditional, but `enforceFocus` allows the focus to stay within the modal, still resetting it otherwise. Thoughts?